integrationtests/src/test/java/io/kroxylicious/proxy/BaseIT.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.proxy;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+import org.apache.kafka.clients.admin.Admin;
+import org.apache.kafka.clients.admin.CreateTopicsResult;
+import org.apache.kafka.clients.admin.DeleteTopicsResult;
+import org.apache.kafka.clients.admin.NewTopic;
+import org.apache.kafka.clients.consumer.Consumer;
+import org.apache.kafka.clients.producer.Producer;
+import org.apache.kafka.common.TopicCollection;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import io.kroxylicious.test.tester.KroxyliciousTester;
+import io.kroxylicious.testing.kafka.junit5ext.KafkaClusterExtension;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@ExtendWith(KafkaClusterExtension.class)
+public abstract class BaseIT {
```

Comment:
What benefit does the abstract class actual bring? All the methods are static so a static final private ctor Utils class might be a less intrusive to users.

Reply:
Yeah, I know what you mean.  Of the two evils, Utils is slight less intrusive IMHO.   I find the imposition of a class hierarchy on my IT quite distasteful.

Reply:
We've had a chat and @gracegrimwood is going to explore adding something to the KroxyliciousTester to take care of this. 

Something along the lines of 

```java
tester.cluster(virtualClusterName).topics().create("abc")
```

With the intent of being useful to filter authors external to the project writing their own integration tests.